src/_gettsim/transfers/rente.py

```diff
@@ -591,43 +566,46 @@ def _referenzalter_abschlag_mit_rente_arbeitsl_frauen(
     if (
         ges_rente_vorauss_langj
         and ges_rente_vorauss_frauen
-        and _ges_rente_vorauss_arbeitsl
+        and ges_rente_vorauss_arbeitsl
     ):
         out = min(
             [
-                ges_rente_frauen_altersgrenze,
+                _ges_rente_frauen_altersgrenze,
                 _ges_rente_langj_altersgrenze,
                 _ges_rente_arbeitsl_altersgrenze,
             ]
         )
     elif ges_rente_vorauss_langj and ges_rente_vorauss_frauen:
-        out = min([ges_rente_frauen_altersgrenze, _ges_rente_langj_altersgrenze])
-    elif ges_rente_vorauss_langj and _ges_rente_vorauss_arbeitsl:
+        out = min([_ges_rente_frauen_altersgrenze, _ges_rente_langj_altersgrenze])
+    elif ges_rente_vorauss_langj and ges_rente_vorauss_arbeitsl:
         out = min([_ges_rente_langj_altersgrenze, _ges_rente_arbeitsl_altersgrenze])
     elif ges_rente_vorauss_langj:
         out = _ges_rente_langj_altersgrenze
     elif ges_rente_vorauss_frauen:
-        out = ges_rente_frauen_altersgrenze
-    elif _ges_rente_vorauss_arbeitsl:
+        out = _ges_rente_frauen_altersgrenze
+    elif ges_rente_vorauss_arbeitsl:
         out = _ges_rente_arbeitsl_altersgrenze
     else:
-        out = float("NaN")
```

Comment:
Similar to above, the reference age for deductions should be the normal retirement age if the individual is not eligible for any other pathway. This is a bit stupid because those individuals cannot claim their pension before that age (and hence there cannot be any deductions) but with this change, the target is much more useful in applications I think.

Reply:
See above, I'll need a bit more explanation first.